allure-generator/src/test/javascript/stubs/handlebars-compiler.js

```diff
@@ -5,8 +5,6 @@ const path = require('path');
 const fs = new MemoryFS();
 const file = process.argv[2];
 
-console.log('SETUP HOHOHO');
-
```

Comment:
This line used to fail tests. Sure, we need to get rid of this and enable JS-tests in the build again.